README.md

```diff
@@ -316,6 +316,27 @@ The renderField mixin can be called in your template with the field to render as
 {{/fields}}
 ```
 
+#### conditionally rendering fields
+
+`renderField` supports conditionally omitting fields if `useWhen` is passed in field config.  `useWhen` accepts another field key `String` and checks the value is `true`, or an `Object` with the keys `field` and `value`.  The field to check cannot appear on the same step - consider using the `toggle` property to show/hide a field on the same step.
+
```

Comment:
Ah ha, do you mean that the field that whose value we depend on, must have been captured on a previous page, i.e., this is not a dynamically updating/displaying field